app/controllers/hunters_controller.rb

```diff
@@ -3,7 +3,7 @@
 # Restful Hunters Controller
 class HuntersController < ApplicationController
   before_action :set_hunter, only: %i[show edit update destroy]
-
+  skip_before_action :authenticate_user!, only: [:index]
```

Comment:
Ooops! Looks like you got a friend there. No worries. You can `rebase -i origin/master` and drop the extra commit.